app/assets/stylesheets/components/editorToolbar.scss

```diff
@@ -0,0 +1,45 @@
+.editor-toolbar {
+  display: flex;
+  background: var(--base-0);
+  padding: var(--su-2) var(--content-padding-x);
+
+  &#overflow-menu {
+    position: asbolute;
+    background: revert;
+    right: var(--content-padding-x);
+    min-width: auto;
+  }
+
+  .formatter-btn {
+    width: 50px;
+
+    svg {
+      vertical-align: middle;
+    }
```

Comment:
Applying `crayons-icon` class to each SVG element should bring all kinds of styling we need to make them look fine-ish.